.github/workflows/ci.yml

```diff
@@ -114,11 +114,11 @@ jobs:
 
         if [[ ${{ matrix.php-version }} == '8.1' ]]; then
           export CODECOVERAGE=1
-          vendor/bin/phpunit --verbose --coverage-clover=coverage.xml
-          CAKE_TEST_AUTOQUOTE=1 vendor/bin/phpunit --verbose --testsuite=database
+          vendor/bin/phpunit tests/TestCase/Database --verbose --coverage-clover=coverage.xml
+          CAKE_TEST_AUTOQUOTE=1 vendor/bin/phpunit tests/TestCase/Database --verbose --testsuite=database
```

Comment:
Do we need to revert this change before merging?

Reply:
Not yet, I will revert it when I getting to updating the ORM once the changes to database package are done.